activation/activation.go

```diff
@@ -176,9 +185,11 @@ func (b *Builder) loop() {
 				err := b.PublishActivationTx(epoch)
 				if err != nil {
 					if err == tooSoonErr {
```

Comment:
While this error is desired for our tests, it is not inline with the protocol.
There should be no reason to prevent someone from publishing an _early_ ATX, if they want to do so.
I think we should have a comment in the code that we should remove this condition once ready (once poet running time is calibrated with epoch length, and probably weights/ticks as well)